src/main/java/io/cdap/plugin/salesforce/plugin/source/batch/SalesforceInputFormat.java

```diff
@@ -74,7 +76,20 @@ public RecordReader createRecordReader(InputSplit split, TaskAttemptContext cont
     return new SalesforceRecordReaderWrapper(sObjectName, sObjectNameField, getDelegateRecordReader(query, schema));
   }
 
-  private RecordReader<Schema, Map<String, ?>> getDelegateRecordReader(String query, Schema schema) {
+  public static RecordReader createInitedRecordReader(
```

Comment:
"Inited" sounds weird. Do you mean `initialized`? Why not call this `createRecordReader`?